rust/azure_iot_operations_services/src/azure_device_registry.rs

```diff
@@ -6,7 +6,7 @@
 use core::fmt::Debug;
 use std::collections::HashMap;
 
-use azure_iot_operations_mqtt::control_packet::QoS as rumqttc_qos;
+use azure_iot_operations_mqtt::control_packet::QoS as mqtt_qos;
 use azure_iot_operations_mqtt::interface::AckToken;
 use azure_iot_operations_protocol::{common::aio_protocol_error::AIOProtocolError, rpc_command};
 use thiserror::Error;
```

Comment:
Make sure to keep our dependencies separate between std library, third party and azure with a line break between the sections so cargo fmt doesn't undo it.

Here and everywhere.

Reply:
It is already separated